src/main/java/com/teragrep/rlp_07/Config.java

```diff
@@ -0,0 +1,53 @@
+package com.teragrep.rlp_07;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class Config {
+    private static final Logger LOGGER = LoggerFactory.getLogger(Config.class);
+    private static int port;
+    public int getPort() {
+        return port;
+    }
+    private static boolean isTls;
+    public boolean isTls() {
+        return isTls;
+    }
+    private static String keystorePassword;
+    public String getKeystorePassword() {
+        return keystorePassword;
+    }
+    public Config() {
+        try {
+            port = Integer.parseInt(System.getProperty("port", "1601"));
+        }
+        catch(NumberFormatException e) {
+            throw new RuntimeException("Can't parse port: " + e.getMessage());
+        }
+        isTls = Boolean.parseBoolean(System.getProperty("tls", "false"));
+        keystorePassword = System.getProperty("tlsKeystorePassword", "changeit");
+    }
+
+    public InputStream getKeystoreStream() throws IOException {
```

Comment:
this is poperational code and should not be part of the config as config should not contain executable code.